src/components/db.ts

```diff
@@ -110,8 +110,9 @@ const initUserProfileTable = async (db: Database): Promise<void> => {
             term VARCHAR(2),
             year INTEGER,
             last_updated TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
is there a current date thing?

Reply:
Yup, I'll use that instead since this is user-facing so people probably don't care about the exact timestamp